PluginGuide.md

```diff
@@ -159,7 +159,7 @@ For this function to work properly, you need to add a `GazableObject` component
 The `Velocity Source` parameter of the `GazableObject` specify where and how the calculation of object velocity should be performed:
 - `Late Update`: this is the most common case. Use this if you modify the position of your object in the Unity `Update` function.
 - `Fixed Update`: Use this if you modify the position of your object in the Unity `FixedUpdate` function.
-- `Rigitbody`: Use this if the position of your object is coming from a rigitbody.
+- `Rigidbody`: Use this if the position of your object is coming from a rigitbody.
```

Comment:
Thank you for finding this.

Since we're fixing the first instance of the typo, let's also fix the 2nd one. If you find any others as well that is helpful.

Happy to merge it after that.

Reply:
Sure. Will fix it. Thanks